src/jit/gentree.h

```diff
@@ -4873,6 +4882,9 @@ struct GenTreePutArgStk : public GenTreeUnOp
     unsigned gtNumSlots;             // Number of slots for the argument to be passed on stack
     unsigned gtNumberReferenceSlots; // Number of reference slots.
     BYTE*    gtGcPtrs;               // gcPointers
+#ifdef _TARGET_ARM_
+    bool gtIsHfa; // isHfa
+#endif
```

Comment:
The comment here is not useful (being a restatement of the member name). Is there something more descriptive that would be useful?